pkgs/development/interpreters/perl-cross/default.nix

```diff
@@ -0,0 +1,54 @@
+{ lib, stdenv, buildPackages, fetchurl }:
+
+let common = {perlVersion, perlSha256}:
+  stdenv.mkDerivation rec {
+    name = "perl-cross";
+    version = perlVersion;
+    crossVersion = "1.1.7";
+
+    srcs = [
+      (fetchurl {
+        url = "mirror://cpan/src/5.0/perl-${perlVersion}.tar.gz";
+        sha256 = perlSha256;
+      })
+
+      (fetchurl {
+        url = "https://github.com/arsv/perl-cross/releases/download/${crossVersion}/${name}-${crossVersion}.tar.gz";
+        sha256 = "0ps1x7wxi3b52igywhx4d6xyr26s87aqdjvkgsnjgxdmcvbyk75p";
+      })
+    ];
+    sourceRoot = "perl-${perlVersion}";
+
+    nativeBuildInputs = [ buildPackages.stdenv.cc ];
+
+    postUnpack = "cp -R perl-cross-${crossVersion}/* perl-${perlVersion}";
+
+    configurePlatforms = [ "build" "host" "target" ];
+
+    enableParallelBuilding = false;
+
+    passthru.libPrefix = "lib/perl5/site_perl";
+
+    meta = {
+      homepage = https://arsv.github.io/perl-cross;
+      description = "Cross-compilation standard implementation of the Perl 5 programmming language";
```

Comment:
This makes it sound like a separate implementation of Perl. But it's only a configure script + Makefiles to be used with standard Perl.

How about this, taken from the their github page: "Configure and build perl with a cross-compiling toolchain".